arch/x86/kvm/vmx/vmx.c

```diff
@@ -2781,7 +2793,7 @@ int setup_vmcs_config_common(struct vmcs_config *vmcs_conf,
 	vmcs_conf->vmentry_ctrl        = _vmentry_control;
 	vmcs_conf->misc	= misc_msr;
 
-#if IS_ENABLED(CONFIG_HYPERV)
+#if IS_ENABLED(CONFIG_HYPERV) && !defined(__PKVM_HYP__)
```

Comment:
nit: we might not need this as this code is already wrapped in `#ifndef __PKVM_HYP__` ?